stronghold/src/account/mod.rs

```diff
@@ -132,48 +128,12 @@ impl Account {
         &self.id
     }
 
-    pub fn mnemonic(&self) -> &String {
```

Comment:
Removing that is fine?

Reply:
the mnemonic was just shifted down, it's still there